core/templates/dev/head/components/side_navigation_bar/side_navigation_bar_directive.html

```diff
@@ -89,35 +89,35 @@
 
   <ul>
     <li ng-class="{'active': NAV_MODE === 'library'}">
-      <a href="/library">
+      <a href="/library" class="protractor-mobile-test-library-link">
         <i class="material-icons oppia-sidebar-menu-icon">&#xE88A;</i>
```

Comment:
Can the class just be `protractor-test-library-link`? Are these links in the side navigation bar not clickable on desktop?

Reply:
Ditto for the other newly added protractor classes!

Reply:
We do not have a side nav bar in desktop. It's only for mobile.

Reply:
Are you sure? I am pretty sure if you resize your Desktop browser to a small enough width, the sidenav will show up, no?

Reply:
Wait, I think you're getting this wrong. I will definitely not be running the site on a desktop with the width of a mobile screen? Isn't this what distinguishes mobile and desktop -- the browser width?

Reply:
Sounds good :)

Reply:
After a close inspection , I would like to drop my review comments about the `protractor-test-mobile` name for the sidenav link. 
The sidenav is not expected to be tested from Desktop 's e2e. We can things as is here!